client/packages/system/src/Stock/ListView/ListView.tsx

```diff
@@ -57,34 +54,35 @@ const StockListComponent: FC = () => {
   const pagination = { page, first, offset };
   const t = useTranslation('inventory');
   const { data, isLoading, isError } = useStock.line.list(queryParams);
-  const [repackId, setRepackId] = React.useState<string | null>(null);
+  // const [repackId, setRepackId] = React.useState<string | null>(null);
   const pluginColumns = usePluginColumns<StockLineRowFragment>({
     type: 'Stock',
   });
-  const EditStockLineCell = <T extends StockLineRowFragment>({
-    rowData,
-    isDisabled,
-  }: CellProps<T>): React.ReactElement<CellProps<T>> => (
-    <IconButton
-      label={t('button.repack')}
-      height="16px"
-      disabled={isDisabled}
-      icon={
-        <StockIcon
-          sx={{
-            color: 'primary.main',
-            width: '12px',
-            cursor: 'pointer',
-          }}
-        />
-      }
-      onClick={e => {
-        e.stopPropagation();
-        repackModalController.toggleOn();
-        setRepackId(rowData.id);
-      }}
-    />
-  );
+  // TO-DO: Move to Stock page
```

Comment:
I think a lot of this commented stuff could be removed?

Reply:
Yeah, have got rid of it on the next PR. Just left it here for now in case I wanted to use some it (which I have). I won't merge this one until I've got them both ready anyway.

Reply:
Awesome thanks 🙏